### PR TITLE
create components for account and implement composite pattern

### PR DESCRIPTION
**Pull request by Cxristian**
create classes for account

### CODE REVIEW

File: src/main/java/pivotaltracker/ui/components/AccountMenu.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * Gets name Account.
+     *
+     * @return a string.
+     */
+    public String getNameAccount() {
+        return nameAccountLbl.getText();
+    }
+}
```
Comment (titor-oopart) [inline src/main/java/pivotaltracker/ui/components/AccountMenu.java]: add line

File: src/main/java/pivotaltracker/ui/pages/account/NotificationsPage.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    /**
+     * Waits until page object is loaded.
+     */
+    @Override
+    protected void waitUntilPageObjectIsLoaded() {
+        wait.until(ExpectedConditions.visibilityOf(notificationsForm));
+    }
+
+}
```
Comment (titor-oopart) [inline src/main/java/pivotaltracker/ui/pages/account/NotificationsPage.java]: add line

File: src/main/java/pivotaltracker/ui/pages/account/ProfilePage.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    private WebElement profileForm;
+
+    /**
+     * Waits until page object is loaded.
+     */
+    @Override
+    protected void waitUntilPageObjectIsLoaded() {
+        wait.until(ExpectedConditions.visibilityOf(profileForm));
+    }
+}
```
Comment (titor-oopart) [inline src/main/java/pivotaltracker/ui/pages/account/ProfilePage.java]: add line

File: src/test/java/steps/AccountsBaseStep.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+    /**
+     * Open page of selected in tab of account.
+     */
+    @When("^I go to the Account in Account page$")
+    public void goToTheContactsPage() {
+        accountBasePage = new AccountBasePage();
+        accountPage = accountBasePage.getAccountNavBar().goToAccountPage();
+    }
+}
```
Comment (titor-oopart) [inline src/test/java/steps/AccountsBaseStep.java]: add line

File: src/test/java/steps/AccountSteps.java [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        logs.info("The Account Page is tested if it owns to the account created");
+        assertEquals(accountPlansPage.getAccountMenu().getNameAccount(), account.getNameAccount());
+    }
+
+    @Then("I should see the new account in list of Accounts page")
+    public void displayTheNewAccountInTheAccountsPage() {
+        logs.info("The Account Page is tested if show in the account page");
+        assertTrue(accountPage.isDisplayedNewAccount(account.getNameAccount()),"the account Name not displayed");
+    }
+}
```
Comment (aburgosu) [inline src/test/java/steps/AccountSteps.java]: Add Line.

**Review by LimbertVargas — APPROVED** [review]
It's ok for me

**Pull request merged**